packages/dashboards/addon/components/dashboard-filters-collapsed-filter.js

```diff
@@ -71,6 +73,17 @@ export default Component.extend({
     return getWithDefault(SUPPORTED_OPERATORS, `${op}.longName`, op);
   }),
 
+  /**
+   * Which id field to use as ID display.
+   */
+  filterValueFieldId: computed('filter.{dimension.name,field}', function() {
+    const meta = this.bardMetadata.getById('dimension', get(this, 'filter.dimension.name'));
```

Comment:
Do we need to use the `get` fn in this method?

Reply:
don't have to, but it does make dealing with null properties much simpler.